scripts/README.md

```diff
@@ -2,25 +2,81 @@
 
 This directory contains utility scripts for the SIGNAL web app.
 
-## Admin Setup
-
-Requires Node 16+. To check, run the following command.
-```bash
-$ node -v
-```
+## Table of Contents
+1. [Tools Setup](#tools-setup)
+2. [Setup Signal](#setup-signal)
+3. [Set up for running the admin scripts](#set-up-for-running-the-admin-scripts)
+4. [Run the upload script](#upload-data)
 
+## Tools Setup
+1. [Download Visual Studio Code](https://code.visualstudio.com/) or use whichever code editor you'd prefer.
+2. [Download Node version 16+](https://nodejs.org/en/download/)
+    - To check if you have node installed, open a terminal window and run the following command.
+    ```bash
+    node -v
+    ```
+3. For MacOS, install xcode command line tools
+   - Check if you have it already installed. In your terminal run the following command.
+     ```
+     xcode-select -p
+     ```
+     If you see an error, the package has not been installed otherwise you're good to go.
+   - To install xcode command line tools run the following command.
+     ```
+     xcode-select --install
+     ```
+4. Install Firebase Tools
+   - Run the following command in terminal
+     ```
+     sudo npm install -g firebase-tools
+     ```
+     
+## Setup Signal
+1. In terminal, navigate to the folder you want to put the `signal-app` project into.
```

Comment:
I might mention command line for the Windows users. I think all the commands are the same except `ls` is `dir`